ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh

```diff
@@ -64,12 +61,10 @@ function check_idp () {
     # Fetch the detailed identityProviders configuration if any
     # Don't quote the $TARGET_RESOURCE variable because it may include spaces
     current_idp_config=$(oc get $TARGET_RESOURCE -o jsonpath='{range '$IDP_FIELD'[*]}{.name}{" "}{.type}{"\n"}{end}')
-    set +x
     if [ -n "$current_idp_config" ] && [ "$current_idp_config" != "null" ] && [ -n "$USERS" ]; then
         echo -e "Skipping addition of new htpasswd IDP because already configured IDP as below:\n$current_idp_config"
         exit 0
     fi
-    set -x
 }
 
 function set_proxy () {
```

Comment:
This is the so-called "some sensitive lines may be not noticed". Under `set -x`, `source "${SHARED_DIR}/proxy-conf.sh"` prints proxy sensitive info, but this was not noticed.